zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/cmd/CRUD.java

```diff
@@ -89,4 +89,12 @@ public static CRUD grantUserInstanceCreateReadUpdateDelete(GetPermissionsRequest
         info().email(session).log("granting full CRUD instance permissions to user");
         return new CRUD().permit(CREATE, READ, UPDATE, DELETE);
     }
+
+    public static CRUD grantServiceAccountInstanceCreateReadUpdateDelete(GetPermissionsRequest request,
+                                                                        ServiceAccount serviceAccount) {
+        info().serviceAccountID(serviceAccount.getId())
```

Comment:
In some places we have ID (datasetID) and others Id (getId), think we should go for 'ID'